pkg/operator/complete.go

```diff
@@ -24,7 +24,7 @@ func verifyResource(cr *imageregistryv1.Config, p *parameters.Globals) error {
 	}
 
 	names := map[string]struct{}{
-		imageregistryv1.ImageRegistryName + "-default-route": {},
+		"default-route": {},
```

Comment:
default-route is used several times in this pull request, should it be a const somewhere instead of using a string every time? (would also help with building the default-route-openshift-image-registry in the tests)

Reply:
updated